dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -199,6 +199,15 @@ def authenticate
         PartialRegistration.persist_attributes(session, user)
         session[:user_return_to] = destination_url
         if DCDO.get('lti_account_linking_enabled', false)
+          metadata = {
+            'user_type' => user.user_type,
```

Comment:
`user_type` is actually automatically added to the custom_ids object https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/lib/metrics/events.rb#L61.

This was functionality was added relatively recently, I still need to go remove all the places where I added it to the metadata.